File: constants/coordinates_and_radius.py
"""Returns a map of coordinates and radius."""
def get_map_of_coordinates_and_radius(lat, lon, radius):
    
    #Coordinates and radius of specified place at the University of Virgina
    map_of_coordinates_and_radius = {
        "coordinates" : {
            "lattitude" : lat,
            "longitude" : lon,
        },
        "radius" : radius,
    }

    return map_of_coordinates_and_radius

File: constants/coordinates_and_radius.ts
/** Returns a map of coordinates and radius. */
export function getMapOfCoordinatesAndRadius(lat: number, lon: number, radius: number): { coordinates: { latitude: number, longitude: number }, radius: number } {
    // Coordinates and radius of specified place at the University of Virginia
    const mapOfCoordinatesAndRadius = {
        "coordinates": {
            "latitude": lat,
            "longitude": lon,
        },
        "radius": radius,
    };

    return mapOfCoordinatesAndRadius;
}
export function getMapOfCoordinates() {
    throw new Error('Function not implemented.');
}

